docs/WindowsToolchain.md

```diff
@@ -0,0 +1,65 @@
+# Windows Toolchain
+
+## Nomenclature
+
+Different terms are used to reference the different portions of the tree.  These help identify the various uses for the content in the tree.
+
+* **Runtime** (`%ProgramFiles%\Swift\Runtimes\0.0.0\...`)
+
+This is the *runtime* component. It contains the libraries which are needed by the user to execute code writtenin Swift. These set of libraries are required to be redistributed with any Swift application or must be provided by the Operating System (e.g. as with macOS). This usually contains the Swift runtime, standard library. It may optionally require Foundation if it is used by the application. Only one copy of the runtime component is required, though each application may bundle its own requisite subset.
```

Comment:
writtenin -> written in
These set -> This set
are required to be -> must be
runtime, standard -> runtime and standard